tensorboard/webapp/webapp_data_source/tb_server.data_source.module.ts

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {NgModule} from '@angular/core';
+import {HttpClientModule} from '@angular/common/http';
+import {TBServerDataSource} from './tb_server.data_source';
+
+@NgModule({
+  imports: [HttpClientModule],
+  providers: [TBServerDataSource],
+})
+export class TBServerDataSourceModule {}
```

Comment:
nit: This is not in a complete state, right? If so, add TODO items to remind us what needs to be added?

Reply:
This feels like more less complete. What needs to be done?

Reply:
OK. I see. The business logic is in the `class TBServerDataSource` in `tb_server.data_source.ts`. 

Resolve this question.